.github/workflows/main.yml

```diff
@@ -123,6 +175,6 @@ jobs:
         run: pnpm build -- --api="http://127.0.0.1:9080"
         shell: bash
 
-      - name: Test
-        run: pnpm test -- --api="http://127.0.0.1:9080"
-        shell: bash
+      - name: Test Packages
+        run: pnpm test -- --filter=./integration-tests/${{matrix.folder}}
+    
```

Comment:
We should keep the `shell: bash` here. This is the reason why we were getting the error `cannot find prisma` with the previous seed setup.